src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -177,6 +177,30 @@ public async Task Reinvite(string orgId, string id)
         await _organizationService.ResendInviteAsync(orgGuidId, userId.Value, new Guid(id));
     }
 
+    [HttpPost("{organizationUserId}/accept-init")]
+    public async Task AcceptInit(Guid orgId, Guid organizationUserId, [FromBody] OrganizationUserAcceptInitRequestModel model)
+    {
+        var user = await _userService.GetUserByPrincipalAsync(User);
+        if (user == null)
+        {
+            throw new UnauthorizedAccessException();
+        }
+
+        if (string.IsNullOrWhiteSpace(model.Key) || model.Keys == null)
+        {
+            throw new BadRequestException();
+        }
```

Comment:
This should already be checked by the `[Required]` attribute on the request model.